infra/label_evaluator_step_functions/handlers/final_aggregate.py

```
@@ -172,6 +173,59 @@ def handler(event: dict[str, Any], _context: Any) -> dict[str, Any]:
+            timestamp_now = datetime.now(timezone.utc)
+            timestamp_str = timestamp_now.strftime("%Y%m%d-%H%M%S")
+            viz_metadata = {
+                "version": timestamp_str,
+                "execution_id": execution_id,
+                "total_receipts": total_receipts,
+                "receipts_with_issues": total_issues > 0
+                and sum(
+                    1
+                    for r in receipt_results
+                    if isinstance(r, dict) and r.get("issues_found", 0) > 0
+                )
```

Ruff (0.15.4)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)